src/extensions/default/PhpTooling/main.js

```diff
@@ -107,7 +107,7 @@ define(function (require, exports, module) {
         CodeHintManager.registerHintProvider(chProvider, ["php"], 0);
         ParameterHintManager.registerHintProvider(phProvider, ["php"], 0);
         CodeInspection.register(["php"], {
-            name: Strings.PHP_DIAGNOSTICS,
```

Comment:
Have something here to differentiate from any other provider which might also provide "php" related linting.

Reply:
As discussed we are not having any name for default PHP Lint Provider